packages/components/src/components/dialog/dialog.tsx

```diff
@@ -164,34 +187,10 @@ const Dialog = ({
     );
 
     if (portal_element_id) {
-        return ReactDOM.createPortal(dialog, document.getElementById(portal_element_id));
+        return ReactDOM.createPortal(dialog, document.getElementById(portal_element_id)!); // non-null assertion
```

Comment:
can we check if  document.getElementById(portal_element_id) exists in the if condition instead of disabling non - null assertion . Something like-
```
if (portal_element_id && document.getElementById(portal_element_id)!==null){
..
 }
```